File: articles/goal-setting-performance-systems/fortran/goal_system_score.f90
! Goal Setting and Performance Systems
! Simple Fortran scoring demonstration.
!
! Responsible-use scope:
! Synthetic-data research, methods demonstration, institutional learning,
! and reproducible workflows only. Not for employment, productivity ranking,
! goal-compliance scoring, or individual decisions.

program goal_system_score_program
  implicit none
  real :: goal_clarity, goal_challenge, feedback_quality, employee_commitment
  real :: strategic_alignment, overload_pressure, metric_distortion
  real :: incentive_misalignment, review_quality, proxy_drift
  real :: goal_feasibility, score

  goal_clarity = 84.0
  goal_challenge = 78.0
  feedback_quality = 82.0
  employee_commitment = 79.0
  strategic_alignment = 83.0
  overload_pressure = 22.0
  metric_distortion = 18.0
  incentive_misalignment = 16.0
  review_quality = 82.0
  proxy_drift = 15.0
  goal_feasibility = 80.0

  score = 0.13 * goal_clarity + 0.10 * goal_challenge + &
          0.13 * feedback_quality + 0.11 * employee_commitment + &
          0.13 * strategic_alignment + 0.11 * review_quality + &
          0.11 * goal_feasibility - 0.07 * overload_pressure - &
          0.06 * metric_distortion - 0.05 * incentive_misalignment - &
          0.04 * proxy_drift

  print *, "Synthetic goal-system effectiveness score:", score
  print *, "Responsible-use reminder: institutional learning only."
end program goal_system_score_program

File: articles/goal-setting-performance-systems/c/goal_system_score.c
/*
Goal Setting and Performance Systems
Simple C scoring demonstration.

Responsible-use scope:
Synthetic-data research, methods demonstration, institutional learning,
goal-system review, performance-system education, and reproducible workflows
only. Not for employee screening, hiring, promotion, compensation, discipline,
termination, workplace surveillance, individual performance management,
productivity ranking, goal-compliance scoring, cultural-loyalty scoring,
or psychological assessment.
*/

#include <stdio.h>

double goal_system_effectiveness_score(
    double goal_clarity,
    double goal_challenge,
    double feedback_quality,
    double employee_commitment,
    double strategic_alignment,
    double overload_pressure,
    double metric_distortion,
    double incentive_misalignment,
    double review_quality,
    double proxy_drift,
    double goal_feasibility
) {
    return
        0.13 * goal_clarity +
        0.10 * goal_challenge +
        0.13 * feedback_quality +
        0.11 * employee_commitment +
        0.13 * strategic_alignment +
        0.11 * review_quality +
        0.11 * goal_feasibility -
        0.07 * overload_pressure -
        0.06 * metric_distortion -
        0.05 * incentive_misalignment -
        0.04 * proxy_drift;
}

int main(void) {
    double score = goal_system_effectiveness_score(84, 78, 82, 79, 83, 22, 18, 16, 82, 15, 80);
    printf("Synthetic goal-system effectiveness score: %.2f\n", score);
    printf("Responsible-use reminder: institutional learning only.\n");
    return 0;
}
